dallinger/deployment.py

```diff
@@ -87,9 +95,13 @@ def handle_launch_data(url, error, delay=DEFAULT_DELAY, attempts=MAX_ATTEMPTS):
             )
         time.sleep(delay)
 
-    error("Experiment launch failed, check server logs for details.")
+    error("Experiment launch failed after multiple attempts.")
```

Comment:
This is a good start. I think we can improve it somewhat by updating `handle_launch_data` so that it knows what the deployment context is and can give specialised advice for each situation. e.g. on Heroku, we should say that the user should go to the Papertrail logs.

Reply:
No doubt, absolutely! Never hesitate to complain.

Reply:
Where do you think the `elif context == "local"` case make sense? You should be already looking at the console if debugging locally...

Reply:
Added some test